R/pkg/R/schema.R

```diff
@@ -123,6 +123,7 @@ structField.character <- function(x, type, nullable = TRUE) {
   }
   options <- c("byte",
                "integer",
+               "float",
```

Comment:
Ok. I added it.

It is ok to create a DataFrame with `float` type. Inserting data from RDD to the DataFrame is no problem too. But if you want to insert local data from R to the DataFrame, it will crash because we serialize `double` in R to `Double` in JVM.